lua/damagelogs/client/rdm_manager.lua

```diff
@@ -697,16 +697,20 @@ function Damagelog:ReportWindow(found, deathLogs, previousReports, currentReport
 		buttons[1]:OnMousePressed()
 	end
 
-	local Logs = vgui.Create("DListView")
-	Logs:AddColumn(TTTLogTranslate(GetDMGLogLang, "Time")):SetFixedWidth(40)
-	Logs:AddColumn(TTTLogTranslate(GetDMGLogLang, "Type")):SetFixedWidth(40)
-	Logs:AddColumn(TTTLogTranslate(GetDMGLogLang, "Event"))
-
-	if deathLogs then
-		self:SetListViewTable(Logs, deathLogs, false)
-	end 
+	if (Damagelog.User_rights[LocalPlayer():GetUserGroup()] or 2) >= 2 then
+		local Logs = vgui.Create("DListView")
+		Logs:AddColumn(TTTLogTranslate(GetDMGLogLang, "Time")):SetFixedWidth(40)
+		Logs:AddColumn(TTTLogTranslate(GetDMGLogLang, "Type")):SetFixedWidth(40)
+		Logs:AddColumn(TTTLogTranslate(GetDMGLogLang, "Event"))
+
+		if deathLogs then
+			self:SetListViewTable(Logs, deathLogs, false)
+		else
+			self:AddLine(TTTLogTranslate(GetDMGLogLang, "Nothinghere"))
```

Comment:
This produces an error since `self` is not the DListView but ApplySchemeSettings.
`Logs:AddLine("", "", TTTLogTranslate(GetDMGLogLang, "Nothinghere"))` is working and the text in the right column.

Reply:
My bad, I added this line after I've done the testing and I just assumed this would work fine.